exercises/etl/canonical-data.json

```diff
@@ -10,7 +10,10 @@
         "such that keys are integers. e.g. in JavaScript, it might look ",
         "like `transform( { 1: ['A'] } );`"
       ],
-      "description": "transforms the a set of scrabble data previously indexed by the tile score to a set of data indexed by the tile letter",
+      "description": "Transform",
```

Comment:
Consider 
```suggestion
      "description": "Transform score based index to letter based index",
```

Reply:
Okay, a single word is a bit too minimal a description, but your suggestion for an identifier leaves my mind twisted and feeling its more a comment than a descriptor.  I committed an alternative that hopefully meets you in the middle.